shared/general/persistent-storage.md

```diff
@@ -19,6 +18,23 @@ When using named volumes, note that:
 - If a device is moved to a new application, the old `/data` folder will be automatically purged.
 - During the build process, data added to a container directory that is configured to link to a named volume will be copied to the volume the first time it's created on the device.
 
+#### Using a Supervisor with a version >= v10.0.0
+
+Since v10.0.0 of balena-supervisor, volumes are no longer
+removed when an application no longer has a reference to
```

Comment:
This section `, volumes are no longer
removed when an application no longer has a reference to...` seems clunking and I struggle to understand it, is there a better way to rephrase it.

Reply:
I thought it was a bit clunky too, I put a suggestion on